src/calendarmanager.cpp

```diff
@@ -224,12 +224,13 @@ QString NemoCalendarManager::getNotebookColor(const QString &notebookUid) const
 
 void NemoCalendarManager::cancelAgendaRefresh(NemoCalendarAgendaModel *model)
 {
-    mAgendaRefreshList.removeOne(model);
+    if (!mAgendaRefreshList.isEmpty())
+        mAgendaRefreshList.removeOne(model);
```

Comment:
Well, the QList documentation says that isEmpty should be checked before calling member functions, see last paragraph of this section: http://doc.qt.io/qt-5/qlist.html#details